docs/updating-the-kit.md

```diff
@@ -1,17 +1,96 @@
 # Updating the kit
 
+There are two ways you could upgrade the prototype kit. If you're confident in your use of the command line interface such as terminal then read the 'Upgrading via the command line' instructions. If you'd prefer not to or are not using the command line to manage your prototype then read the 'Upgrading manually' instructions. 
```

Comment:
We've changed from 'updating' to 'upgrading' - probably best to stick to one throughout